client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -77,6 +78,40 @@ class ThemeEnhancements extends Component {
 	}
 
 	renderInfiniteScrollSettings() {
+		if ( this.props.jetpackSettingsUI ) {
+			return this.renderJetpackInfiniteScrollSettings();
+		}
+		return this.renderSimpleSiteInfiniteScrollSettings();
+	}
+
+	renderSimpleSiteInfiniteScrollSettings() {
+		const { translate } = this.props;
+		return (
+			<FormFieldset>
+				<FormLegend>{ translate( 'Infinite Scroll' ) }</FormLegend>
+
+				<div className="theme-enhancements__info-link-container site-settings__info-link-container">
+					<InfoPopover position="left">
+						<ExternalLink
+							href="https://support.wordpress.com/...infinite-scroll" /* @TODO fix this URL */
+							icon
+							target="_blank" /* @TODO rel noopener, etc */
+						>
+							{ translate( 'Learn more about Infinite Scroll.' ) }
+						</ExternalLink>
+					</InfoPopover>
+				</div>
+
+				{ this.renderToggle(
+					'infinite_scroll',
+					false,
+					translate( 'Load posts as you scroll. Disable to show a clickable button to load posts' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Check to load posts as you scroll. Uncheck to show clickable button to load posts' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).